src/tuxedo-tomte

```diff
@@ -47,6 +47,8 @@ my $shareDir = '/usr/share/tuxedo-tomte/';
 my $reposList = $shareDir.'repolist.txt';
 my $distribution;
 my $distributionVersion;
+my $distributionIdLike;
+my $distributionVersionCodename;
```

Comment:
New variables to save ID_LIKE and VERSION_CODENAME of /etc/os-release in function readOSData and to use them in function workaroundUbuntuForks